docs/source/index.rst

```diff
@@ -164,7 +165,8 @@ Please see our `contributing guidelines <https://github.com/cleanlab/cleanlab/bl
    cleanlab/outlier
    cleanlab/multiannotator
    cleanlab/multilabel_classification/index
-   cleanlab/token_classification/index
+   cleanlab/regression/index
+   cleanlab/token_classification/indexs
```

Comment:
indexs?

Reply:
Unsure how this typo happened, but fixed!